core-web/apps/dotcms-ui/src/app/view/components/dot-theme-selector-dropdown/dot-theme-selector-dropdown.component.ts

```diff
@@ -83,6 +83,18 @@ export class DotThemeSelectorDropdownComponent
                 /* */
             }
         }, 0);
+
+        // Here we set the initial value of the dropdown as System Theme
+        this.paginatorService.url = 'v1/themes';
+        this.paginatorService.paginationPerPage = 5;
+        this.paginatorService.setExtraParams('hostId', 'SYSTEM_HOST');
```

Comment:
Don't we have to contemplate the scenario where the system host should not be present at all ?

Reply:
@fmontes Can this happen? What should we do in this case?

Reply:
No, system host is system and is hardcoded in the backend